<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="full_menu_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="logo_label_2">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resource.qrc">:/icons/icons/Logo.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="logo_label_3">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>StockWise</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="home_btn_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Home</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icons/icons/home-4-32.ico</normaloff>
             <normalon>:/icons/icons/home-4-48.ico</normalon>:/icons/icons/home-4-32.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="dashboard_btn_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Dashboard</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icons/icons/dashboard-5-32.ico</normaloff>
             <normalon>:/icons/icons/dashboard-5-48.ico</normalon>:/icons/icons/dashboard-5-32.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="order_btn_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Orders</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icons/icons/product-32.ico</normaloff>
             <normalon>:/icons/icons/product-48.ico</normalon>:/icons/icons/product-32.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="customer_btn_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Customers</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icons/icons/activity-feed-32.ico</normaloff>
             <normalon>:/icons/icons/activity-feed-48.ico</normalon>:/icons/icons/activity-feed-32.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>98</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_butt_2">
         <property name="text">
          <string>Exit</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/icons/icons/close-window-64.ico</normaloff>:/icons/icons/close-window-64.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>14</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="menu">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icons/icons/menu-4-32.ico</normaloff>:/icons/icons/menu-4-32.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>488</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <widget class="QWidget" name="page">
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>150</y>
             <width>71</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>home</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_2">
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>130</y>
             <width>191</width>
             <height>71</height>
            </rect>
           </property>
           <property name="text">
            <string>dashboard</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_3">
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>140</y>
             <width>111</width>
             <height>91</height>
            </rect>
           </property>
           <property name="text">
            <string>order</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_4">
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>150</y>
             <width>39</width>
             <height>11</height>
            </rect>
           </property>
           <property name="text">
            <string>customer</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="icon_only_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="logo_label_1">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resource.qrc">:/icons/icons/Logo.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="home_btn_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icons/icons/home-4-32.ico</normaloff>
             <normalon>:/icons/icons/home-4-48.ico</normalon>:/icons/icons/home-4-32.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="dashboard_btn_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icons/icons/dashboard-5-32.ico</normaloff>
             <normalon>:/icons/icons/dashboard-5-48.ico</normalon>:/icons/icons/dashboard-5-32.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="order_btn_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icons/icons/product-32.ico</normaloff>
             <normalon>:/icons/icons/product-48.ico</normalon>:/icons/icons/product-32.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="customer_btn_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icons/icons/activity-feed-32.ico</normaloff>
             <normalon>:/icons/icons/activity-feed-48.ico</normalon>:/icons/icons/activity-feed-32.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>353</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_butt_1">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/icons/icons/close-window-64.ico</normaloff>:/icons/icons/close-window-64.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_only_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>full_menu_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>home_btn_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>home_btn_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dashboard_btn_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>dashboard_btn_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>order_btn_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>order_btn_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customer_btn_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>customer_btn_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>home_btn_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>home_btn_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dashboard_btn_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>dashboard_btn_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>order_btn_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>order_btn_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customer_btn_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>customer_btn_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_butt_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_butt_1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
